app/src/main/java/de/westnordost/streetcomplete/quests/AbstractOsmQuestForm.kt

```diff
@@ -119,6 +119,10 @@ abstract class AbstractOsmQuestForm<T> : AbstractQuestForm(), IsShowingQuestDeta
 
         setTitle(getString(osmElementQuestType.getTitle(element.tags)))
         setTitleHintLabel(getNameAndLocationSpanned(element, resources, featureDictionary))
+        val objNote = element.tags["note"]
+        if (objNote != null) {
+            setObjNote(objNote)
+        }
```

Comment:
```suggestion
        setObjNote(element.tags["note"])
```
No need for null check here, as null will be handled by setObjNote

Reply:
Resolved with afcb138